include/stdlib.h

```diff
@@ -158,6 +158,21 @@ _PDCLIB_PUBLIC int atexit( void (*func)( void ) );
 */
 _PDCLIB_PUBLIC void exit( int status );
 
+/* Register a function that will be called on quick_exit().
+   At least 32 functions can be registered this way, and will be called in
+   reverse order of registration (last-in, first-out).
+   Returns zero if registration is successfull, nonzero if it failed.
+*/
+_PDCLIB_PUBLIC int at_quick_exit( void (*func)( void ) );
+
+/* Quick process termination. Functions registered by at_quick_exit() (see above) are
```

Comment:
Not sure if respected on this repo: 80 column limit

Reply:
Fixed.